thrift/src/main/java/com/linecorp/armeria/server/thrift/ThriftDocServicePlugin.java

```diff
@@ -402,6 +405,17 @@ static TypeSignature toTypeSignature(FieldValueMetaData fieldValueMetaData) {
         return TypeSignature.ofUnresolved(firstNonNull(unresolvedName, "unknown"));
     }
 
+    @VisibleForTesting
+    static EnumInfo newEnumInfo(Class<? extends Enum<? extends TEnum>> enumType) {
+        requireNonNull(enumType, "exceptionClass");
```

Comment:
I guess we could remove `requireNonNull` if this method is called only one place with non-null variable and `package-private`. 😀